include/dlaf/lapack/tile.h

```diff
@@ -469,6 +572,11 @@ void potrf(cusolverDnHandle_t handle, const blas::Uplo uplo, const matrix::Tile<
   auto info = potrfInfo(handle, uplo, a);
   assertExtendInfo(dlaf::cusolver::assertInfoHegst, handle, std::move(info));
 }
+
+template <class T>
+void stedc(cusolverDnHandle_t, const Tile<BaseType<T>, Device::CPU>&, const Tile<T, Device::CPU>&) {
+  DLAF_STATIC_UNIMPLEMENTED(sizeof(T) == 0);
```

Comment:
I was just browsing and wanted to add a reminder to change this:

```suggestion
  DLAF_STATIC_UNIMPLEMENTED(T);
```

I haven't looked closely at this PR otherwise.